keyboards/kopibeng/mnk65_stm32/info.json

```
@@ -81,22 +978,21 @@
                 {"matrix": [3, 9], "x": 9.25, "y": 3},
                 {"matrix": [3, 10], "x": 10.25, "y": 3},
                 {"matrix": [3, 11], "x": 11.25, "y": 3},
-                {"matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3},
+                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                 {"matrix": [3, 13], "x": 14, "y": 3},
                 {"matrix": [3, 14], "x": 15, "y": 3},
-
-                {"matrix": [4, 0], "w": 1.25, "x": 0, "y": 4},
-                {"matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4},
-                {"matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4},
-                {"matrix": [4, 4], "w": 2.25, "x": 3.75, "y": 4},
-                {"matrix": [4, 6], "w": 1.25, "x": 6, "y": 4},
-                {"matrix": [4, 8], "w": 2.75, "x": 7.25, "y": 4},
-                {"matrix": [4, 10], "w": 1.25, "x": 10, "y": 4},
-                {"matrix": [4, 11], "w": 1.25, "x": 11.25, "y": 4},
+                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 4], "x": 3.75, "y": 4, "w": 2.25},
+                {"matrix": [4, 6], "x": 6, "y": 4, "w": 1.25},
+                {"matrix": [4, 8], "x": 7.25, "y": 4, "w": 2.75},
+                {"matrix": [4, 10], "x": 10, "y": 4, "w": 1.25},
+                {"matrix": [4, 11], "x": 11.25, "y": 4, "w": 1.25},
                 {"matrix": [4, 12], "x": 13, "y": 4},
                 {"matrix": [4, 13], "x": 14, "y": 4},
                 {"matrix": [4, 14], "x": 15, "y": 4}
             ]
         }
     }
-}
+}
```

add EoF (it's a GitHub thing)
```suggestion
}

```